app/routes/campaignRoute.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import CampaignController from '../controllers/campaignController';
+
+const router = express.Router();
+const campaignController = new CampaignController();
+
+
+function initRoutes() {
+  router.get('/clients', (req, res) => {  
+    campaignController.getClients(req, res);
+  });
+
+  router.get('/clients/:id', (req, res) => {
+    campaignController.getClientById(req, res);
+  });
+
+  router.post('/clients', (req, res) => {
+    campaignController.createClient(req, res);
+  });
+
+  router.delete('/clients/:id', (req, res) => {
+    campaignController.deleteClient(req, res);
+  });
+
+  router.get('/clients/:id/lists', (req, res) => {
+    campaignController.getList(req, res);
+  });
+
+  return router;
+}
+
+module.exports = initRoutes;
```

Comment:
why to use module.exports? In ES6 we can simply use export keyword.
export default .....